java/test3/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Shell script to build and run a Java program
+# tied to test/datalog_tests/span_string.dl program
+
+set -ex
+
+if command clang -v 2>/dev/null; then
+    export CC=clang
+else
+    export CC=gcc
+fi
+
+case `uname -s` in
+    Darwin*)    SHLIBEXT=dylib; JDK_OS=darwin;;
+    Linux*)     SHLIBEXT=so; JDK_OS=linux;;
+    *)          echo "Unsupported OS"; exit -1
+esac
+
+# Compile the span_uuid.dl DDlog program
+ddlog -i ../../test/datalog_tests/span_string.dl -j -L../../lib
+# Compile the rust program; generates ../test/datalog_tests/span_ddlog/target/release/libspan_ddlog.a and .so
+pushd ../../test/datalog_tests/span_string_ddlog
+cargo build --release
+popd
+# Build the Java library with the DDlog API
+make -C ..
+# Force linking with the static library by deleting the dynamic library
+rm -f ../../test/datalog_tests/span_string_ddlog/target/release/libspan_string_ddlog.so
```

Comment:
This should no longer be needed, as `ddlog` now has flags to explicitly control static/dynamic lib generation:
```
           --dynlib         generate dynamic library
           --no-dynlib      do not generate dynamic library (default)
           --staticlib      generate static library (default)
           --no-staticlib   do not generate static library
```

so only the static lib will be generated by default